### ISSUE TITLE
Improve Image element

### ISSUE THREAD

**Issue by chjj** [chjj/blessed#102]
The image element has some quirks right now. It's usually a cell or two off of its position and originally renders in the wrong location. This may be blessed's fault or w3mimgdisplay's. While Images are definitely usable, some debugging is probably necessary here.

**Comment by piranna** [comment]
Another problem it has is that needs a framebuffer device or X11 and don't work on pure text-mode terminals. blessed-contrib has a Picture element that draw images as ascii-art, maybe blessed Image element would implement something similar as a fall-back for text-mode terminals and only use w3mimgdisplay for graphic terminals

**Comment by chjj** [comment]
Terminals cells aren't going to come close to displaying detailed images. I would rather just have a kind of canvas element with helper functions which easily draw cells to the screen buffer. This is already possible by simply modifying the screen buffer.

Images have been fixed with d69ed5c2943c8c439dbf38324441511983ea7ea5 among other commits. On top of that, I also added a `screen.displayImage` method, which simply opens w3m to display and image if the user wants.
